package.json

```diff
@@ -48,13 +48,7 @@
   },
   "dependencies": {
     "chokidar": "^3.6.0",
-    "gulp": "^5.0.0",
-    "gulp-cli": "^3.0.0",
+    "braces": "^3.0.3",
```

Comment:
Is this just a hack for the dependency issue?

Reply:
how so? for braces?

Reply:
yeah - is braces an explicit dependency or just a dependency of something else that we are now adding here to avoid using an earlier version?

Reply:
hm i tried deleting braces from both `node_modules` and the `package.json`. I deleted the package-lock and ran `npm i` again, and it still showed up in the modules and `package-lock.json`. Does that mean it is a dependency of another dependency, so doesn't need to be explicitly stated as a direct dependency in `package.json`? The version installed was `3.0.3` (newest)

Reply:
Yeah that seems right. Then it doesn't need to be listed.